server/routes/ad.ts

```diff
@@ -479,10 +477,19 @@ export default class AdService {
     opensearchDashboardsResponse: OpenSearchDashboardsResponseFactory
   ): Promise<IOpenSearchDashboardsResponse<any>> => {
     try {
+      var { resultIndex } = request.params as { resultIndex: string };
+      if (
+        !resultIndex ||
+        !resultIndex.startsWith(CUSTOM_AD_RESULT_INDEX_PREFIX)
+      ) {
+        resultIndex = '';
```

Comment:
by specifying the string as `''`, does this indicate no custom result index is specified? If so can you add this in comments?

Reply:
done.